### PR TITLE
Add in a menu based rag example to use in docs.

### PR DESCRIPTION
**Pull request by schnecle**

### CODE REVIEW

File: js/samples/menu-example/rag/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ */
+
+import { configureGenkit } from '@genkit-ai/core';
+import { devLocalVectorstore } from '@genkit-ai/dev-local-vectorstore';
+import { defineFlow, runFlow } from '@genkit-ai/flow';
+import { textEmbeddingGecko, vertexAI } from '@genkit-ai/vertexai';
+import * as z from 'zod';
+import { indexMenu } from './indexer';
+
+export default configureGenkit({
```
Comment (MichaelDoyle) [inline js/samples/menu-example/rag/src/index.ts:24]: Does this need to be exported?

Comment (tagboola) [reply js/samples/menu-example/rag/src/index.ts]: Nope, removed.

**Review by tagboola — APPROVED** [review]

File: js/samples/menu-example/rag/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Build it
+
+```
+pnpm build
+```
+
+or if you need to, build everything:
+
+```
+cd ../../../; pnpm build; pnpm pack:all; cd -
```
Comment (MaesterChestnut) [inline js/samples/menu-example/rag/README.md:12]: is there a better way we can say "go to root" instead of '../../../ ?

**Review by MaesterChestnut — APPROVED** [review]

File: js/samples/menu-example/rag/README.md (hunk 2 above)
Comment (tagboola) [reply js/samples/menu-example/rag/README.md:12]: Not sure, but we actually use it in a bunch of samples e.g. byo-evaluator, cat-eval, evaluator-gut-check so atleast it's consistent 🤷

**Pull request merged**